src/assets/styles/styles/custom/_autocomplete.scss

```diff
@@ -1,39 +1,67 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public 
  * License, v. 2.0. If a copy of the MPL was not distributed with this 
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+.more-left-margin {
+  margin-left: -6px !important;
+}
+
 .listbox {
   position: absolute;
-  margin: 0;
-  padding: 10px;
+  padding-top: 10px;
+  padding-left: 0px;
+  padding-bottom: 10px;
+  margin-left: -2px;
+  margin-top: -5px;
   z-index: 1;
-  background-color: $white;
   list-style: none;
   overflow: auto;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell,
-    "Open Sans", "Helvetica Neue", sans-serif;
-  font-size: medium;
+  font-family: Roboto;
```

Comment:
Przydałoby się skorzystać tutaj ze zmiennej. Podobnie jak w innych miejscach - wszystkie fonty, kolory, wielkości czcionek, im więcej tym lepiej. Zwłaszcza jeśli coś już wcześniej było używane. Pozwoli nam to na zachowanie spójnego designu.